lib/ansible/modules/dnf.py

```diff
@@ -198,6 +198,13 @@
     type: bool
     default: "no"
     version_added: "2.10"
+  nobest:
+    description:
+      - Set best option to False, so that transactions are not limited to best candidates only.
+    required: false
+    type: bool
+    default: "no"
+    version_added: "2.10"
```

Comment:
If it's going to ship with Ansible 2.11, should I change this to `version_added: "2.11"`?

Reply:
yes

Reply:
Eventually this will need to change to 2.11. We're in feature freeze for 2.10 currently, so new features cannot be merged. If you change this to 2.11 now, the sanity test will fail. You could go ahead and do that and we can just re-run tests once we branch for the 2.10 release and update the sanity test.

Reply:
I've changed to 2.11, please tell me if I should do anything else until shipping 2.10 and updating the sanity test.